### ISSUE TITLE
Message without reply_markup does not work

### ISSUE THREAD

**Issue by jasperkuperus** [jehy/telegram-test#3]
If I send a message without `reply_markup`, e.g. just `/start`, the promise never seems to resolve. Or I don't understand it yet, or this line (81) doesn't do what it's supposed to:

```
} else resolve(self.sendUpdate(chatId));// wait until some available actions appear
```

**Comment by jehy** [comment]
If there is no `reply_markup`, it means that we have no available keyboard actions. So we have nothing to return. So we wait until we receive some `reply_markup` from another answer on user message - once we receive it, promise will be resolved. That's just as planned - something wrong about it?

**Comment by jasperkuperus** [comment]
Ah. Ok. My bot will not return `reply_markup` at all, just a message which I want to test. So by waiting on a message with `reply_markup`, my simple message is skipped and never resolved.

**Comment by jehy** [comment]
Ok, I'll think what I can do. This should really be tested, not skipped.

**Comment by jehy** [comment]
Yup, I expected that developers will only test bot replies which contain action choises, but that was wrong. Fixed in version 0.1.4.
